Common/Helpers/BlobStorageManager.cs

```diff
@@ -10,15 +10,15 @@ namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers
     /// <summary>
     /// Helper methods, related to blob storage.
     /// </summary>
-    public class BlobStorageHelper : IBlobStorageHelper
+    public class BlobStorageManager : IBlobStorageManager
     {
-        private readonly string _connectionString;
-        private readonly string _containerName;
+        private readonly ICloudBlobContainerProvider _blobContainerProvider;
 
-        public BlobStorageHelper(string connectionString, string containerName)
+        public BlobStorageManager(string connectionString, string containerName)
```

Comment:
So this guy essentially exposes the CloudBlobContainer back out for consumers to make direct calls on the CloudBlobContainer? Wouldn't it be better for the BlobStorageManager to expose methods that interact with the CloudBlobContainer and making it more testable for consumers?